src/woo_search/search_index/api/serializers/search.py

```diff
@@ -0,0 +1,74 @@
+from django.utils.translation import gettext_lazy as _
+
+from drf_polymorphic.serializers import PolymorphicSerializer
+from rest_framework import serializers
+
+from ...constants import ResultTypeChoices, SortChoices
+from . import DocumentSerializer, PublicationSerializer
+
+
+class DocumentPolymorphicSerializer(serializers.Serializer):
+    record = DocumentSerializer(read_only=True)
+
+
+class PublicationPolymorphicSerializer(serializers.Serializer):
+    record = PublicationSerializer(read_only=True)
+
+
+class SearchSerializer(serializers.Serializer):
+    query = serializers.CharField(
+        required=False,
+        help_text=_(
+            "Filtering records based on the provided search term."
+            " This field searches the data within the following fields"
+            " (with priority based on the order below):\n\n"
+            "- `identifier` **field only present in Document*\n"
+            "- `officieleTitel`\n"
+            "- `verkorteTitel`\n"
+            "- `omschrijving`"
+        ),
+    )
+    page = serializers.IntegerField(
+        default=1, help_text=_("A page number within the paginated result set.")
+    )
+    page_size = serializers.IntegerField(
+        default=10, help_text=_("Number of results to return per page.")
+    )
+    sort = serializers.ChoiceField(
+        choices=SortChoices.choices,
+        default=SortChoices.relevance,
```

Comment:
If no query is supplied, what happens when sorting by relevance?

Reply:
Then it will use the ES `score` value (which can be influenced by multiple inputs besides `query`) with a fallback on `laatst_gewijzigd_datum`.